app/bundles/LeadBundle/Resources/views/Auditlog/_table.html.twig

```diff
@@ -0,0 +1,67 @@
+<table class="table">
+    <tr>
+        <th>Field/Object</th>
+        <th>New Value</th>
+        <th>Old Value</th>
```

Comment:
It would be nice to have these translatable.

Reply:
added the translations